src/main/java/org/sonarlint/cli/Options.java

```diff
@@ -58,24 +60,38 @@ public static Options parse(String[] args) throws ParseException {
       } else if ("-X".equals(arg) || "--debug".equals(arg)) {
         options.verbose = true;
 
-      } else if ("--report-dir".equals(arg)) {
+      } else if ("--html-report".equals(arg)) {
         i++;
         if (i >= args.length) {
-          throw new ParseException("Missing argument for option --report-dir", i);
+          throw new ParseException("Missing argument for option " + arg, i);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Missing argument for option " 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)